### PR TITLE
kepubify 4.0.0

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
> Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - kepub/test.epub

**Comment by chenrui333** [comment]
test.epub was removed in the upstream repo and we need a dedicated test.epub to test out the formula change. All in all, this `test.epub` in the fixtures would be a good thing for testing out all the formulae.

**Comment by pgaskin** [comment]
Some notes about kepubify v4:
- Please add `-tags zip117` to the build arguments to keep the performance like the release builds.
- Go 1.16 is now the minimum Go version.

**Comment by chenrui333** [comment]
Thanks @EricFromCanada! :)

**Review by SMillerDev — APPROVED** [review]

File: Formula/kepubify.rb [hunk 1]
```diff
@@ -26,18 +26,17 @@ def install
     ].each do |p|
       system "go", "build", "-o", bin/p,
                    "-ldflags", "-s -w -X main.version=#{version}",
+                   "-tags", "zip117",
                    "./cmd/#{p}"
```
Comment (SMillerDev) [inline Formula/kepubify.rb:30]:
```suggestion
      system "go", "build", 
        "-tags", "zip117", 
        *std_go_args(ldflags: "-s -w -X main.version=#{version}"), 
        "-o", bin/p,
        "./cmd/#{p}"
```

**Pull request closed without merging**